util/normalize.lua

```diff
@@ -6,11 +6,15 @@ and add individual player names to recipient list
 --]]
 function mail.normalize_players_and_add_recipients(field, recipients, undeliverable)
     local order = mail.parse_player_list(field)
-    for _, player_name in ipairs(order) do
-        if not minetest.player_exists(player_name) then
-            undeliverable[player_name] = true
-        else
-            recipients[player_name] = true
+    for _, recipient_name in ipairs(order) do
+        if mail.split(recipient_name, "@")[1] == "" then -- in case of maillist
```

Comment:
Use `string.split` instead of `mail.split`.
```suggestion
        if recipient_name:split("@")[1] == "" then -- in case of maillist
```